pipeline.py

```diff
@@ -59,30 +81,71 @@ class TextProcessingPipeline:
     Process articles from corpus manager
     """
     def __init__(self, corpus_manager: CorpusManager):
-        pass
+        self.corpus_manager = corpus_manager
+        self.text_to_process = ''
 
     def run(self):
         """
         Runs pipeline process scenario
         """
-        pass
+        articles = self.corpus_manager.get_articles()
+        for file in articles.values():
+            self.text_to_process = file.get_raw_text()
+            morph_tokens = self._process()
+            processed_text = []
+            for token in morph_tokens:
+                processed_text.append(str(token))
+            file.save_processed(' '.join(processed_text))
 
     def _process(self) -> List[type(MorphologicalToken)]:
         """
         Performs processing of each text
         """
+        mystem_analyser = Mystem()
```

Comment:
are you sure about other keys? why do you check only for analysis key existence? this part is very fragile